### ISSUE TITLE
BUG: Wildcard stopped working on 1.13.0

### ISSUE THREAD

**Issue by sounak-kun** [Flow-Launcher/Flow.Launcher#1910]
### Checks

- [X] I have checked that this issue has not already been reported.

### Problem Description

Suppose there is a file called `hello_world.txt`
Everything supports wildcards in it's search, so searching for `hello*.txt` will find that file.
In 1.12,1 I have used this feature in Flow Launcher search (with everything indexing), so I can confirm this is only an issue in the latest version.

### To Reproduce

1. Create a file called hello_world.txt
2. Rebuild Everything cache and search for `hello*.txt` to ensure the file has been indexed by Everything
3. Type `hello*.txt` in Flow Launcher and see no results.

### Screenshots

![image](https://user-images.githubusercontent.com/93094402/218954850-1bec630e-3611-48d6-b66f-825382169262.png)

### Flow Launcher Version

1.13,0

### Windows Build Number

10.0.22621.1105

### Error Log

No error, thus no logs

**Comment by deefrawley** [comment]
Working fine for me. Are you sure you get a match in Everything when you use the wildcard?

**Comment by sounak-kun** [comment]
@deefrawley Yes. I attached a screenshot of Everything finding a match. Note, using a wildcard at the beginning works. Like `*.jpg` still works. It only does not when I use it in the middle. Like in `a*.jpg` to find all images that start with a.

**Comment by deefrawley** [comment]
Sorry, so you did. Can you post screenshots of your settings for the Explorer Plugin

**Comment by sounak-kun** [comment]
@deefrawley Here are the screenshots to all 5 pages.
![Flow Launcher_Mjx6TmpQRm](https://user-images.githubusercontent.com/93094402/218968864-f491911b-3d9e-45e4-9aa0-fe2aacfb539b.png)
![Flow Launcher_Jy302mbDSO](https://user-images.githubusercontent.com/93094402/218968898-3de87108-c577-48d6-874e-93ce807804f1.png)
![Flow Launcher_KhkubNCaJd](https://user-images.githubusercontent.com/93094402/218968908-3564e102-ec87-4945-a4df-144d68e5876b.png)
![Flow Launcher_RDniDRY6Tx](https://user-images.githubusercontent.com/93094402/218968912-2fd28ddd-dc97-4de8-a276-4c5126c3af97.png)
![Flow Launcher_jo343bIRDt](https://user-images.githubusercontent.com/93094402/218968927-142f7612-c626-4b87-8b54-ea1791afcd33.png)

**Comment by deefrawley** [comment]
Can you add the path to Everything like below (noting to add in the actual install location on your system), restart Flow, and see if that helps

![image](https://user-images.githubusercontent.com/18594055/218975852-20dc09a6-3252-4046-b8f1-31d30dc27d35.png)

**Comment by sounak-kun** [comment]
@deefrawley Tried, same result. Still can't parse wildcards.

**Comment by deefrawley** [comment]
What about the version of Everything?

Also just to show it's working for me (Everything v1.4.1.1022)

![image](https://user-images.githubusercontent.com/18594055/219287964-aeb032e1-bcf9-42fd-9629-3467f6103b66.png)

**Comment by sounak-kun** [comment]
@deefrawley I am also running v1.4.1.1022.

**Comment by sounak-kun** [comment]
I downgraded and I can confirm that the feature is working in 1.12.1
![HUPS76NP9k](https://user-images.githubusercontent.com/93094402/219449815-7727c13c-08dd-4b0f-a615-f75ab2b3dd25.png)
It only stopped working in 1.13.0
![JgmpR2IXhs](https://user-images.githubusercontent.com/93094402/219449920-31a6223f-9d11-4457-8ae4-4f3485c1b3b5.png)

**Comment by jngit23** [comment]
Good afternoon from Brazil.
I suffer from the same problem here, I use Version 22H2 of Windows 11, Build 22623.1325.
I checked like you with FlowLauncher version 1.12.1 and it actually works, analyzing the <Explorer> plugin I noticed its version, 2.2.0 different from the new FlowLauncher which is 3.0.0, so what I did was copy the old version of the plugin in the new FlowLauncher, by new I mean version 1.13.0. Working normally now :D.

![Captura de Tela (65)](https://user-images.githubusercontent.com/107755720/220725315-d25dfdba-c68a-4b45-b62d-8bdfe2c804a0.png)

![Flow Launcher_mwNfOxukor](https://user-images.githubusercontent.com/107755720/220725891-a8ee7e61-6ecc-4092-ab5a-ca197f479894.png)

![Flow Launcher_Os4i5C4zY7](https://user-images.githubusercontent.com/107755720/220726150-e2350845-f1eb-4de6-bd30-3bc0a10392f3.png)
